index.src.html

```diff
@@ -1762,6 +1767,15 @@
   <dd>
     An encoding [=scalability mode identifier=] as defined by [[WebRTC-SVC]].
   </dd>
+
+  <dt><dfn dict-member for=VideoEncoderConfig>bitrateMode</dfn></dt>
+  <dd>
+    Configures encoding to use a {{BitrateMode/constant}} or
+    {{BitrateMode/variable}} bitrate as defined by [[MEDIASTREAM-RECORDING]].
+
+    NOTE: The precise degree of bitrate fluctuation in either mode is
+        implementer defined.
```

Comment:
We usually say "implementation defined".

Reply:
Done.